stress/common/src/main/java/alluxio/stress/jobservice/JobServiceMaxThroughputResult.java

```diff
@@ -11,11 +11,11 @@
 
 package alluxio.stress.jobservice;
 
-import alluxio.stress.common.AbstractMaxThroughputSummary;
+import alluxio.stress.common.AbstractMaxThroughputResult;
 
 /**
  * The summary for the max throughput test.
  */
-public final class JobServiceMaxThroughputSummary
-    extends AbstractMaxThroughputSummary<JobServiceBenchParameters, JobServiceBenchSummary> {
+public final class JobServiceMaxThroughputResult
```

Comment:
IMO it's better to keep the naming as summary since it is an aggregation of all single tests. and it's more aligh with JobServiceBenchSummary and JobServiceBenchTaskResult

Reply:
I refer to this class `ClientIOTaskResult`, which is also the implementation class of TaskResult and Summary.
I think the meaning of `TaskResult` is to aggregate multiple test results (although it is not implemented here, just for `Benchmark` framework) and Summary means to draw results.

Reply:
What do you think about keeping the name of the result. If you still don't feel right, I will change it back~

Reply:
I would prefer to keep original summary and update ClientIO one